recipe/meta.yaml

```
@@ -1,40 +1,49 @@
-{% set version = "1.6.0" %}
+{% set version = "1.10.0" %}
 
 package:
   name: lazy-object-proxy
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/l/lazy-object-proxy/lazy-object-proxy-{{ version }}.tar.gz
-  sha256: 489000d368377571c6f982fba6497f2aa13c6d1facc40660963da62f5c379726
+  sha256: 78247b6d45f43a52ef35c25b5581459e85117225408a4128a3daf8bf9648ac69
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - {{ compiler('c') }}
   host:
     - python
+    - wheel
     - pip
-    - setuptools_scm >=3.3.1
+    - setuptools_scm >=8.0.0
```

```suggestion
    - setuptools
    - setuptools_scm >=8.0.0
```
Missing setuptools, see https://github.com/ionelmc/python-lazy-object-proxy/blob/aa46e846fa2eb7cb845a07e122f2984267551098/pyproject.toml#L3